torch/_dynamo/decorators.py

```diff
@@ -67,6 +68,20 @@ def skip(fn=None):
     return fn
 
 
+def disable_if_config_true(config_attr: str):
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            if getattr(config, config_attr, None):
+                return disable(func)(*args, **kwargs)
```

Comment:
Do we need `disable(recursive=True)` here?

Reply:
it's calling `disable` and `disable` has `recursive=True`. but I can be explicit here with `recursive=True`